code/game/world.dm

```diff
@@ -60,7 +60,19 @@ GLOBAL_VAR(restart_counter)
 
 #ifdef USE_BYOND_TRACY
 #warn USE_BYOND_TRACY is enabled
-	init_byond_tracy()
+	// Ugly having this whole snippet in here but we want it to run ASAP
```

Comment:
Wat?

Reply:
If `RealGlobalsPreInit` is called before *every* static variable (which is the point of the change Lemon is making) then fine but this really should stay as its own proc for readability's sake

Reply:
Oh I thought you wanted it inlined so it could come absolutely first. Good, cause I hate it like this.